xarray/tests/test_backends.py

```diff
@@ -492,6 +492,14 @@ def test_dropna(self):
             actual = on_disk.dropna(dim='x')
             assert_identical(expected, actual)
 
+    def test_ondisk_after_print(self):
+        """ Make sure print does not load file into memory """
+        in_memory = create_test_data()
+        with self.roundtrip(in_memory) as on_disk:
+            if not on_disk['var1']._in_memory:
```

Comment:
Can we write this without the conditional check? It's usually best to avoid control flow in tests -- it makes it more obvious what is going on.